### DISCUSSION TITLE
Help with finding a compatible Huawei E3372 stick

### DISCUSSION THREAD

**Discussion by miaszh** [tesla-android/issue-tracker#234] [General]
I've got the android up on the rpi4 but I still cannot find a compatible LTE Modem which works out of the box with the android\rpi.
Anyone can confirm a version of the Huawei E3372?
I found E3372-**153** (according to #167 it didn't work) or E3372h-**607**

I need help find the compatible one.
Thanks in advance.

**Comment by mikegapinski** [comment]
It should work, just make sure it's in hilink mode and supports all the LTE bands you need
